src/Servers/Kestrel/Core/src/Internal/Http2/FlowControl/InputFlowControl.cs

```diff
@@ -19,94 +19,125 @@ namespace Microsoft.AspNetCore.Server.Kestrel.Core.Internal.Http2.FlowControl;
 /// <seealso href="https://datatracker.ietf.org/doc/html/rfc9113#name-flow-control"/>
 internal sealed class InputFlowControl
 {
-    private readonly int _initialWindowSize;
+    private struct FlowControlState
+    {
+        private const long AbortedBitMask = 1L << 32; // uint MaxValue + 1
```

Comment:
To avoid the comment 😉.
```suggestion
        private const long AbortedBitMask = uint.MaxValue + 1L;
```

Reply:
I was thinking about the same, but somehow it was cleaner to my head where the 1 is. So, I would feel an urge to add a comment with the suggestion `// 1L << 32;`

Reply:
> was cleaner to my head where the 1 is

Makes sense. Keep it as is then.

Reply:
I also prefer the shift plus the comment.